angr/analyses/cfg/cfg_base.py

```diff
@@ -2120,6 +2136,25 @@ def _graph_traversal_handler(self, g, src, dst, data, blockaddr_to_function, kno
     # Other functions
     #
 
+    def _loc_to_funcloc(self, location):
+        """
+
+        :param int location:
+        :return:
+        """
+        return location
```

Comment:
It seems to me that `_loc_to_funcloc()` is useless...

Reply:
True. As in other cases, this is an old function that has been introduced by error during the rebase. I cleaned it now